modules/apps/27-interchain-accounts/host/ibc_module_test.go

```diff
@@ -130,7 +150,7 @@ func (suite *InterchainAccountsTestSuite) TestChanOpenInit() {
 
 	// use chainB (host) for ChanOpenInit
 	msg := channeltypes.NewMsgChannelOpenInit(path.EndpointB.ChannelConfig.PortID, icatypes.Version, channeltypes.ORDERED, []string{path.EndpointB.ConnectionID}, path.EndpointA.ChannelConfig.PortID, icatypes.ModuleName)
-	handler := suite.chainB.GetSimApp().MsgServiceRouter().Handler(msg)
+	handler := suite.chainB.GetSimApp().MsgServiceRouter.Handler(msg)
 	_, err := handler(suite.chainB.GetContext(), msg)
```

Comment:
This differs from one of my earlier tests and looks cleaner.  I will try and modify.

Reply:
I do not know why this doesn't work the same as:

https://github.com/cosmos/ibc-go/pull/1443/files#diff-2114e4419debdc1aea40268325e1ad68ea7a5186f26d34fd1fe310f87a183ea7